sds_data_manager/lambda_code/SDSCode/api_lambdas/download_api.py

```diff
@@ -112,6 +159,13 @@ def lambda_handler(event, context):
             ),
         )
 
+    if not is_authenticated_user(event) and not is_released(filepath):
+        return http_response(
+            status_code=403,
+            body="The requested file is not released yet. Please contact the "
+            "SDC team for more information.",
```

Comment:
I wouldn't tell anyone to contact us, they can reach out themselves if they want. Also include the filepath for help debugging potentially.
```suggestion
            body=f"The file {filepath} is not a part of a public release yet."
```